src/managers/config.py

```diff
@@ -591,24 +595,23 @@ def controller_properties(self) -> list[str]:
         Returns:
             List of properties to be set
         """
-        # TODO: this check will need to be done on relation data
-        roles = "process.roles="
+        if self.state.kraft_mode == False:  # noqa: E712
+            return []
```

Comment:
**note:** After seeing this `== False`/falsey issue enough, I agree, we should raise or find some other solution, allowing this to be a reliable `bool`. Not sure how to do that though